src/actions/channels.js

```diff
@@ -181,6 +182,20 @@ export function createGroupChannel(userIds: Array<string>): ActionFunc {
             last_update_at: created.create_at,
         };
 
+        if (created.create_at !== created.update_at) {
```

Comment:
Is this checking that the GM is not new? Is this always going to work? Feels like a bit of a hacky assumption. Might be a good idea to add a comment stating the assumption

Reply:
Yes, that was the idea, but I think is better to use the `total_msg_count` field, because it detects exactly what i expect. I going to change it and leave a comment anyway.

Reply:
Fixed :)